database/seeders/TestDataSeeder.php

```diff
@@ -19,5 +21,13 @@ public function run()
             'email' => 'test@test.at',
             'password' => bcrypt('test'),
         ]);
+
+        Channel::factory()
```

Comment:
Are you using this new data somewhere?

- I believe this file is a left over which is not used (I need to delete it later)
- I am missing a test for single cal download. Can you add that?

Reply:
Sure will do